src/content/2.2/code/agda/Section212.agda

```diff
@@ -0,0 +1,456 @@
+{- Chapter 12. Limits and Colimits -}
+
```

Comment:
I am not sure if including whole text of the chapters with Agda snippets is a good idea. This increase significantly number of changes to review once here and later when making PR to original repository.